scripts/sanity-checks.sh

```diff
@@ -2,6 +2,11 @@
 
 set -eo pipefail
 
+if [[ ! "${INPUT_SCANNERVERSION}" =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+  echo "::error title=SonarScanner::Invalid scannerVersion format. Expected format: x.y.z.w (e.g., 7.1.0.4889)"
```

Comment:
Not tested, but maybe show the wrong value to help people debugging?
```suggestion
  echo "::error title=SonarScanner::Invalid scannerVersion '${INPUT_SCANNERVERSION}'. Expected format: x.y.z.w (e.g., 7.1.0.4889)"
```